config.yaml

```diff
@@ -0,0 +1,253 @@
+---
+aither:
```

Comment:
Should we nest these under a `trackers` object instead of having all trackers as a top level config item?